src/test/resources/environments.json

```diff
@@ -1,8 +1,9 @@
 {
   "prod": {
     "BASE_URL_API": "http://the-internet.herokuapp.com",
-    "BASE_URL": "http://the-internet.herokuapp.com/",
+    "BASE_URL": "https://www.amazon.in/",
     "IMAGES_BASE_URL": "http://images.google.com/",
-    "BING_BASE_URL": "http://bing.com/images"
+    "BING_BASE_URL": "http://bing.com/images",
+    "BROWSER": "firefox"
```

Comment:
why is BROWSER defined here? It is not picked up from here. Delete it

Reply:
Acknowledge